docs/docs/forms.md

```diff
@@ -360,9 +359,12 @@ class Form extends React.Component {
   render() {
     return (
       <div>
-        <input type="radio" name="choice" value="A" onChange={this.handleChange} /> Option A<br />
-        <input type="radio" name="choice" value="B" onChange={this.handleChange} defaultChecked={true} /> Option B<br />
-        <input type="radio" name="choice" value="C" onChange={this.handleChange} /> Option C<br />
+        <input type="radio" name="choice" value="A"
```

Comment:
This is not very readable. There is also a problem: clicking on the label doesn't select the option. This is not a very good UX.

I also don't think `name` attribute is relevant in this example.

We should use `<label>` instead, like this:

```js
        <label>
          <input type="radio" value="A" onChange={this.handleChange} />
          Option A
        </label>
```